back/src/Services/SocketManager.ts

```diff
@@ -787,14 +789,13 @@ export class SocketManager {
         }
     }
 
-    private notifyStopMeetOnRoomLeave(room: GameRoom) {
+    private notifyStopMeetOnRoomLeave(room: GameRoom, user: User) {
```

Comment:
Should `user` be optional?
Like `private notifyStopMeetOnRoomLeave(room: GameRoom, user?: User) {`